tools/deploy_documentation_tag.sh

```diff
@@ -26,12 +26,12 @@ pwd
 CURRENT_TAG=`git describe --abbrev=0`
 IFS='.'
 read -ra VERSION <<< "$CURRENT_TAG"
-STABLE_VERSION="${VERSION[0]}\.${VERSION[1]}"
+STABLE_VERSION=${VERSION[0]}.${VERSION[1]}
```

Comment:
I get stable version as `0.18.0.` when I try this line locally. 

```suggestion
STABLE_VERSION =`echo $VERSION | cut -d "." -f -2`
```
This is what I used for building the [translated ML docs](https://github.com/qiskit-community/qiskit-translations/blob/master/tools/deploy_ML_translated_documentation.sh#L23)